modules/dhcp/providers/server/isc.rb

```diff
@@ -78,13 +78,10 @@ def parse_config_and_leases_for_records
             opts.merge!(parse_record_options(data))
           end
         end
-        begin
-          subnet = @service.find_subnet(opts[:ip])
-          next unless subnet
-          ret_val << Proxy::DHCP::Reservation.new(opts.merge(:subnet => subnet))
-        rescue Exception => e
-          logger.warn "skipped #{hostname} - #{e}"
-        end
+        next if !opts.has_key?(:ip)
```

Comment:
Would you also mind adding a test (if you do the above) to check that the entry isn't showing up in the subnet data?